CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [0.0.41] - 2021-02-29
```

Comment:
I don't think you add the date when simply updating the code. I think that's reserved for when you're merging develop into master

Reply:
ah right, good to know, I can remove it. When should this get released into master?

Reply:
Whenever you really require it to, but yeah it does have to be a separate action to making code updates